src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -80,6 +80,76 @@ public function prototype($type)
     }
 
     /**
+     * Shorthand for prototype('variable').
```

Comment:
I think so, I think that the method name is self explanatory. But wouldn't break the code standards check?